utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -452,6 +452,7 @@ function Set-DeploymentExamplesSection {
                 '  params: {'
                 ($bicepParamsArray | ForEach-Object { "  $_" }),
                 '  }'
+                '}'
```

Comment:
I guess it needs to run on the newly added authorization locks as well. Could you do that?

Reply:
I sure can.

Reply:
done